docs/tutorial.md

```diff
@@ -0,0 +1,183 @@
+# Oria Operator Tutorial
+
+### An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.8.0 or newer
+- User authorized with `cluster-admin` permissions.
```

Comment:
I would probably add a reason why they should have cluster-admin. Something like "... permissions so that you can install an operator, CRDs and change rbac". 

That will give the user an idea of why. And they might be really advanced and choose to do it without clusteradmin because they have enough privileges based.

Reply:
updated